File: TASK-03/ SubtasK 2/Main.java
import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Paths;

public class FileCopy {
    public static void main(String[] args) {
   
        String inputFilePath = "input.txt";
        String outputFilePath = "output.txt";

        try {
            
            String content = new String(Files.readAllBytes(Paths.get(inputFilePath)));

           
            Files.write(Paths.get(outputFilePath), content.getBytes());

            System.out.println("File content copied successfully.");
        } catch (IOException e) {
            System.err.println("An error occurred: " + e.getMessage());
        }
    }
}

File: TASK-03/ SubtasK 2/main.cpp
#include <iostream>
#include <fstream>
#include <string>

int main() {
   
    std::string inputFilePath = "input.txt";
    std::string outputFilePath = "output.txt";

 
    std::ifstream inputFile(inputFilePath);
    if (!inputFile) {
        std::cerr << "Error opening input file: " << inputFilePath << std::endl;
        return 1;
    }

    
    std::string content((std::istreambuf_iterator<char>(inputFile)), std::istreambuf_iterator<char>());
    inputFile.close(); 

  
    std::ofstream outputFile(outputFilePath);
    if (!outputFile) {
        std::cerr << "Error opening output file: " << outputFilePath << std::endl;
        return 1;
    }

    /
    outputFile << content;
    outputFile.close(); 

    std::cout << "Content successfully copied from " << inputFilePath << " to " << outputFilePath << std::endl;

    return 0;
}
